core/services/relay/evm/event_binding.go

```diff
@@ -130,16 +105,27 @@ func (e *eventBinding) GetLatestValue(ctx context.Context, params, into any) err
 }
 
 func (e *eventBinding) QueryKey(ctx context.Context, filter query.KeyFilter, limitAndSort query.LimitAndSort, sequenceDataType any) ([]commontypes.Sequence, error) {
-	remappedFilter, err := e.remapFilter(filter)
+	remapped, err := e.remap(filter)
 	if err != nil {
 		return nil, err
 	}
```

Comment:
if !e.bound {
		return fmt.Errorf("%w: event not bound", commontypes.ErrInvalidType)
	}

remapped, err := e.remap(filter)
	if err != nil {
		return nil, err
	}